contracts/NFTStorefront.cdc

```diff
@@ -320,18 +385,26 @@ pub contract NFTStorefront {
         init (
             nftProviderCapability: Capability<&{NonFungibleToken.Provider, NonFungibleToken.CollectionPublic}>,
             nftType: Type,
+            nftUUID: UInt64,
             nftID: UInt64,
             salePaymentVaultType: Type,
             saleCuts: [SaleCut],
-            storefrontID: UInt64
+            storefrontID: UInt64,
+            customID: String,
+            commissionAmount: UFix64,
```

Comment:
why do we have both `saleCuts` and `commissionAmount`?

Reply:
+1 it seems like a commission could be a sales cut.

otoh, if the idea is to bake-in/enforce a commission across multiple marketplaces (that might have different listing tx's), should the commission be created more like a salesCut (`commissions: [Commission]`) so it might be distributed across multiple parties?

Reply:
Yeah, it is up to the marketplace places to honour them, Royalties are always a voluntary thing so we can't enforce it.